include/mrdox/Metadata/Symbols.hpp

```diff
@@ -109,7 +109,6 @@ using OptionalSymbolID = Optional<SymbolID>;
 */
 enum class InfoKind
 {
-    Default,
     Namespace,
```

Comment:
This needs to explicitly name the 0 value for bitstream. And you should let `Function` have the value 0, because it is the most common.